scripts/lib/installer-win32-gen.tsx

```diff
@@ -221,16 +282,23 @@ export default async function generateFileList(rootPath: string): Promise<string
     </ComponentGroup>
 
     {descendantDirs.map(d => <ComponentGroup Id={d.id} Directory={d.id}>
```

Comment:
Are we setting both `Id` and `Directory` to `d.id`? or should `Directory` be set to `d.name`?

Reply:
The `Id` attribute needs to be unique among components, which is why it's set to `d.id` (it has a `d_` prefix, so it's kind of namespaced there).

The `Directory` attribute needs to refer to the `Id` attribute of the `<Directory>`, hence it's set to `d.id`.